pkg/cmd/roachtest/django.go

```diff
@@ -206,7 +206,7 @@ func registerDjango(r *testRegistry) {
 		MinVersion: "v19.2.0",
 		Name:       "django",
 		Owner:      OwnerAppDev,
-		Cluster:    makeClusterSpec(1, cpu(8)),
+		Cluster:    makeClusterSpec(1, cpu(16)),
```

Comment:
I don't see an explicit setting for memory, hope this works. Have you been able to test?

Reply:
yeah, it seems like a non-ideal way of specifying, but it works because this will make it request the `n1-standard-16` instance type. this is how other tests work too (e.g. `roachtest/tpcc.go`)